cfme/configure/settings.py

```diff
@@ -13,18 +13,26 @@
 from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 
 
+# Todo: Will remove this widget from here once it is available in widgetastic_patternly.
+class SaveButton(Button):
+    @property
+    def disabled(self):
+        return 'disabled' in self.browser.classes(self)
```

Comment:
Does the Button not handle disabled?

Reply:
No the current button doesn't work for it, I have update the button widget, it's PR is about to get merge, will update this PR accordingly.